### PR TITLE
1603: Fix issue can not save welcome page logo

### LINKED ISSUE #1603: Unable to change Welcome Image
From the `Welcome` tab of the admin pages, the Game Designer can change the welcome image logo:
image

When I try to add an SVG or PNG image, I get this error in the node console:
image

### PR DESCRIPTION
**Pull request by lebaphi**
Fixes #1603 

https://user-images.githubusercontent.com/26705621/174212502-91ea082c-171f-4fe9-b1dc-135acca1aefd.mov

### CODE REVIEW

File: packages/server/server.js [hunk 1]
```diff
@@ -149,15 +149,6 @@ const runServer = (
     fs.writeFile(image, req.body, err => console.log(err))
 
     res.status(200).send({ path: imagePath })
-    // const buff = Buffer.from(req.body, 'utf8').toString()
```
Comment (IanMayo) [inline packages/server/server.js:152]: @lebaphi - could you explain why these lines are commented out please?

Shouldn't we just delete them?

Comment (lebaphi) [reply packages/server/server.js:152]: I don't comment out them @IanMayo, I've seen it here before and I just removed them to clean up the code

Comment (IanMayo) [reply packages/server/server.js:152]: Sorry - I misinterpreted the PR.  Yes, I see you removed them :-)

**Review by IanMayo — APPROVED** [review]
Working fine, thanks.

**Pull request merged**